tests/scenario_aeat347.rst

```diff
@@ -314,3 +307,30 @@ Reassign 347 lines::
     >>> invoice.reload()
     >>> invoice.aeat347_operation_key
     'A'
+
+Create out invoice an empty leave::
```

Comment:
A banda del wizard estem fent el mateix que el test que has retocat més amunt, no¿?

"Create out invoice over limit, but changing manually the operation key"